ts/Core/Axis/TreeGridAxis.ts

```diff
@@ -479,15 +487,25 @@ namespace TreeGridAxis {
 
                     // Update yData now that we have calculated the y values
                     axis.series.forEach(function (series: Highcharts.Series): void {
-                        var data = (series.options.data || []).map(function (
+                        var axisData = (series.options.data || []).map(function (
                             d: Highcharts.PointOptionsType
                         ): Highcharts.PointOptionsType {
+
+                            if (series.options.keys && series.options.keys.length) {
+                                // Get the axisData from the data array used to
+                                // build the treeGrid where has been modified
+                                data.forEach(function (point: Highcharts.GanttPointOptions): void {
+                                    if ((d as any).includes(point.x) && (d as any).includes(point.x2)) {
```

Comment:
Two things here:
- `includes` is not supported by IE at all
- what if `d: PointOptionsType` is object type? Or in general, not an array?